public/content/documentation/native/controls/chip.md

```diff
@@ -0,0 +1,214 @@
+## General Notes
+
+How to test a chip
+
+## Videos
+
+### Andriod Talkback
+<video controls>
+  <source src="media/video/native/chip/chip-AndroidTalkback.mp4" type="video/webm">
+  Your browser does not support the video tag.
+</video>
+
+## Condensed
+
+### #a11y - Native Accessibility Acceptance Criteria
+
+How to test a chip
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Tab, arrow keys or ctl+tab: Focus visibly moves to the chip
+   - Space: Any elements inside are activated on iOS and Android
+   - Enter: Any elements inside are activated on Android
+2. Test mobile screenreader gestures
+
+   - Swipe: Focus moves to the element, expresses its state, if applicable
+   - Doubletap: Activates the chip
+
+3. Listen to screenreader output on all devices
+
+   - Name: Purpose is clear and matches the group visible label or chip label
+   - Role: Identifies as a button in iOS and "double tap to activate" or checkbox in Android
+   - Group: n/a
+   - State: Expresses its state (selected/disabled/dimmed)
+
+4. Test device settings
+
+   - Text resize: Text can resize up to 200% without losing information
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/chip](https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/chip)
+
+## Gherkin
+
+### #a11y - Native Accessibility Acceptance Criteria
+
+How to test a chip
+
+GIVEN THAT I am on a screen with a chip
+
+1. Scenario: Test keyboard actions
```

Comment:
The gherkin spacing here is different than the dropdown gherkin spacing (though similar to dropdown gherkin scenario 1?).